astore/server/astore/retrieve.go

```diff
@@ -76,15 +79,14 @@ func (s *Server) Retrieve(ctx context.Context, req *astore.RetrieveRequest) (*as
 			return nil, status.Errorf(codes.InvalidArgument, "Invalid path - %s", err)
 		}
 	} else {
-		query = datastore.NewQuery(KindArtifact).Order("-Created")
+		query = datastore.NewQuery(KindArtifact)
```

Comment:
uhm .. doesn't this break the order in which artifacts are returned?

Reply:
We expect exactly one artifact in a query with no LIMIT clause, so order must not matter?